### PR TITLE
Move `ARCH` and `nps` to `worker_info`.

### PR DESCRIPTION
**Pull request by vdbergh**
In preparation for the db conversion we do some more cleanups.

### CODE REVIEW

**Comment by vdbergh** [comment]
This is also server.

**Comment by ppigazzini** [comment]
"worker" label : "issues requiring (also) a worker update"

**Comment by vdbergh** [comment]
> "worker" label : "issues requiring (also) a worker update"

Ah ok :)

File: server/fishtest/templates/tests_view.mak [hunk 1]
```diff
@@ -311,14 +311,19 @@ from fishtest.util import worker_name
...
+                  ARCH = task.get('ARCH', '?')
             %>
             % if 'worker_info' in task:
                 os: ${task['worker_info']['uname']};
                 ram: ${task['worker_info'].get('max_memory', '?')}MiB;
                 gcc: ${gcc_version};
                 python: ${python_version};
                 worker: ${version};
-                arch: ${task.get('ARCH', '?')}
+                arch: ${ARCH},
```
Comment (ppigazzini) [inline server/fishtest/templates/tests_view.mak:326]: Is the comma a pasto?

Comment (vdbergh) [reply server/fishtest/templates/tests_view.mak:326]: Yep...

Comment (vdbergh) [reply server/fishtest/templates/tests_view.mak]: Fixed.

**Comment by ppigazzini** [comment]
A worker on master gets this message:
```
Error from remote: /api/request_task: request['worker_info']['ARCH'] is missing
```

**Comment by vdbergh** [comment]
> A worker on master gets this message:
> 
> ```
> Error from remote: /api/request_task: request['worker_info']['ARCH'] is missing
> ```
> 
> EDIT_000: after triggering the workers update perhaps this state has low/null probability to happen

Yes this is expected. The error is harmless (no state on the server is changed) and will trigger a worker update.

**Review by ppigazzini — APPROVED** [review]
Ok, thank you. The PR looks good on DEV.

**Pull request merged**